patches/000-initial-setup.patch

```diff
@@ -193,15 +265,64 @@ index 0000000000..c0800df578
 +	x := (*(*[]big.Word)(unsafe.Pointer(&b)))[:len(b)]
 +	return new(big.Int).SetBits(x)
 +}
+diff --git a/src/crypto/internal/backend/boringtest/config.go b/src/crypto/internal/backend/boringtest/config.go
+new file mode 100644
+index 0000000000..976aa85cea
+--- /dev/null
++++ b/src/crypto/internal/backend/boringtest/config.go
+@@ -0,0 +1,43 @@
++/* Test configuration package for OpenSSL FIPS
++
++The FIPS mode behavior of OpenSSL varies between versions and distributions
++depending which version of the FIPS standard the library targets. Because
++the Go crypto tests can not reliably account for these behavioral differences, 
++building golang-fips on a new distribution often results in test failures due to
++variations in things like supported crypto algorithms and key sizes.
++
++The goal of this package is to implement a compile-time defined configuration
++for the behavior of OpenSSL, which is more easily configurable to run in different
++environments.  The compile-time schema was chosen as the preferred method, because
++we don't want elements of the run-time environment to impact the result of the tests
++(for example, changes to the environment or config files).
++*/
++
++package boringtest
++
++import (
++	"testing"
++)
++
++var TestConfig map[string]bool
```

Comment:
I don't think we need to export this, all callers seem to use the already exported `Supports()` function.